SurfnPaddleRails/app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ActiveRecord::Base
+
+  validates_presence_of :title, :body, :summary, :author_id, :published
```

Comment:
Try validating the presence of `author` rather than `author_id` ... That way you validate not just that there's a value in `author_id` but that said value maps to an actual author.